utilities/source_utilities.py

```diff
@@ -411,7 +411,7 @@ def delete_events_in_building_block(objectId_list_to_delete):
         event = find_one(current_app.config['EVENT_COLLECTION'], condition= _id)
         url = current_app.config['EVENT_BUILDING_BLOCK_URL'] + '/' + str(event.get('platformEventId'))
         result = requests.delete(url, headers=headers)
-        if result.status_code != 202:
+        if result.status_code != 202 or result.status_code != 404:
```

Comment:
@bingzhang, I think this needs to be an AND operation as only if these two statuses are seen, we will be deleting events from the Events Manager local database. Please check. Thanks.

Reply:
yes you are right.